core/src/main/java/org/apache/calcite/runtime/ResultSetEnumerable.java

```diff
@@ -95,29 +95,32 @@
           };
         } else {
           //noinspection unchecked
```

Comment:
is this comment not also obsolete? In my patch I removed it; reason: unchecked cast is gone